extra/changelog-generator/changelog-generator

```diff
@@ -23,7 +23,7 @@ SHA_TO_TRACKER = {}
 
 JIRA_REGEX = r"\[?(?:Jira:? *)?(?:https?://tracker.mender.io/browse/)?((?:INF|ARCHIVE|MEN|QA|MC)(?:-| )[0-9]+)\]?"
 TRACKER_REGEX = r"\(?(?:(?:Ref|Ticket):? *)?%s\)?:? *" % (JIRA_REGEX)
-CONVENTIONAL_COMMIT_TYPE_REGEX = r"^(fix|feat|chore)(?:\(\S+\))?!?: "
+CONVENTIONAL_COMMIT_TYPE_REGEX = r"^(fix|feat|chore)(?:\(\S+\))?!?: ?"
```

Comment:
A small downside to this:
The conventional commit standard states that a colon and space is mandatory after the initial commit keyword. There might be problems (small, but problems nontheless) if we were to adopt any third party conventional commit tooling in the future.

Reply:
I see. We might want to make the opposite change then: Make sure [`check_commits.sh`](https://github.com/mendersoftware/mendertesting/blob/master/check_commits.sh) doesn't accept this. Because right now it accepts it, and then the entry is misrendered in the changelog.

Reply:
100% agree.

Reply:
Agreed!

However, I think we should keep this change then, since there are already commits like that in our history, and this needs to render them correctly. It's not responsible for correctness, it's just responding to what's there.

Reply:
If they already exist then yes. But perhaps amend the commit message of this particular commit for environment friendly purposes.

Reply:
:+1:

Reply:
https://github.com/mendersoftware/mendertesting/pull/206